shell/probe/install.sh

```diff
@@ -5,8 +5,10 @@ PRELUDE_ACCOUNT_ID=""
 PRELUDE_ACCOUNT_SECRET=""
 PROBE_NAME="nocturnal"
 ENDPOINT_ID=$(hostname)
+ENDPOINT_TAGS=""
 ENDPOINT_TOKEN=""
-DOS="$(uname | awk '{print tolower($0)}')-$(uname -m)"
+sys=$(uname | awk '{print tolower($0)}')
```

Comment:
what's this change about?

Reply:
original behavior of this script was to send in OS as a tag. install.sh was hardcoding "darwin", this line change was to account for 'linux' too. Since your other comment was to remove 'windows' as an automatic tag, I'll do the same here and this line change will be reverted

Reply:
ah, thanks!